pkg/addons2/seaweedfs/install.go

```diff
@@ -27,9 +27,15 @@ func (s *SeaweedFS) Install(ctx context.Context, kcli client.Client, writer *spi
 		return errors.Wrap(err, "create prerequisites")
 	}
 
+	airgapChartsPath := ""
+	if s.IsAirgap {
+		airgapChartsPath = runtimeconfig.EmbeddedClusterChartsSubDir()
+	}
+
 	hcli, err := helm.NewHelm(helm.HelmOptions{
 		KubeConfig: runtimeconfig.PathToKubeConfig(),
 		K0sVersion: versions.K0sVersion,
+		AirgapPath: airgapChartsPath,
```

Comment:
why not override the `ChartPath` field below instead?

Reply:
because that has a format like `oci://proxy.replicated.com/anonymous/...` and then I'd have to special case behavior based on the format of the string

Pretty sure the 'pull OCI' code would not appreciate being passed a filepath

Reply:
i agree with above. how about using url scheme `file://`?

Reply:
I've refactored the code to only have to specify the AirgapPath once and then pass the resulting hcli object into each addon - that way the airgap logic doesn't have to be duplicated

Does that make sense to both of you?